src/join.sol

```diff
@@ -79,35 +85,52 @@ contract GemJoin {
     uint    public dec;
     uint    public live;  // Active Flag
 
+    // Events
+    event Rely(address indexed usr);
+    event Deny(address indexed usr);
+    event Join(address indexed urn, uint256 wad);
+    event Exit(address indexed guy, uint256 wad);
```

Comment:
Also here we log `guy` while exit() gets `usr`.

Reply:
I'm thinking this is probably ok, just acknowledge you are aware and I'll resolve and approve.

Reply:
This should be `usr`.

Reply:
Everything updated to `usr` here.